test_driver/transitions_perf.dart

```diff
@@ -5,21 +5,31 @@
 import 'dart:async';
 import 'dart:convert' show JsonEncoder;
 
+import 'package:flutter/foundation.dart' show kIsWeb;
+import 'package:flutter/material.dart';
 import 'package:flutter_driver/driver_extension.dart';
 import 'package:gallery/data/demos.dart';
+import 'package:gallery/main.dart' show GalleryApp;
+
 import 'package:gallery/l10n/gallery_localizations_en.dart';
-import 'package:gallery/main.dart' as gallery;
 
 Future<String> _handleMessages(String message) async {
-  assert(message == 'demoDescriptions');
-
-  final demoDescriptions = allGalleryDemos(GalleryLocalizationsEn())
-      .map((demo) => demo.describe)
-      .toList();
-  return const JsonEncoder.withIndent('  ').convert(demoDescriptions);
+  switch (message) {
+    case 'demoDescriptions':
+      final demoDescriptions = allGalleryDemos(GalleryLocalizationsEn())
+          .map((demo) => demo.describe)
+          .toList();
+      return const JsonEncoder.withIndent('  ').convert(demoDescriptions);
+      break;
+    case 'isWeb':
+      return kIsWeb.toString();
```

Comment:
How is handle messages used? Does it make sense that it can return 'True' or 'False'?

Reply:
It serves as the channel to pass messages from the app with the driver. It can only pass strings, though. I had to do it this way because only the app has access to things like kIsWeb